src/components/MermaidDash.js

```diff
@@ -497,8 +450,10 @@ class MermaidDash extends Component {
     });
 
     const projectSet = [...new Set(projectFilter)];
+
     const memberExtract = choices.reduce((memberList, project) => {
-      if (projectSet.find(id => id === project.id)) memberList.push(...project.members);
+      if (projectSet.find(id => id === project.id) && Array.isArray(project.members))
```

Comment:
Had to add the array check because it kept crashing for some reason. Not sure if it's related to what I'm doing 🤔